drivers/hmis/app/graphql/schema.graphql

```diff
@@ -1548,6 +1573,7 @@ enum DataCollectionFeatureRole {
   CE_ASSESSMENT
   CE_EVENT
   CURRENT_LIVING_SITUATION
+  CUSTOM_CASE_NOTE
```

Comment:
Note a huge deal, but I think it would be better if this was just called `CASE_NOTE`. The "Custom" is a prefix convention for the table name so that we can export it with HUD records, but, in terms of the API surface there is nothing about it that is "custom", its just a case note. I think this name kind of implies that there is a non-custom type of case note, which there's not.

I'd change the definition identifier too (so the json file name, currently) to just be case_note.